packages/scene-composer/src/utils/entityModelUtils/modelRefComponent.ts

```diff
@@ -0,0 +1,96 @@
+import { ComponentRequest, ComponentUpdateRequest } from '@aws-sdk/client-iottwinmaker';
+import { DocumentType } from '@aws-sdk/types';
+
+import { IModelRefComponent, KnownComponentType } from '../../interfaces';
+import { componentTypeToId } from '../../common/entityModelConstants';
+import { IModelRefComponentInternal } from '../../store';
+import { generateUUID } from '../mathUtils';
+
+enum ModelRefComponentProperty {
+  Uri = 'uri',
+  ModelType = 'modelType',
+  UnitOfMeasure = 'unitOfMeasure',
+  LocalScale = 'localScale',
+  CastShadow = 'castShadow',
+  ReceiveShadow = 'receiveShadow',
+}
+
+export const createModelRefComponent = (model: IModelRefComponent): ComponentRequest => {
+  const comp: ComponentRequest = {
+    componentTypeId: componentTypeToId[KnownComponentType.ModelRef],
+    properties: {},
+  };
+  if (model.uri) {
+    comp.properties![ModelRefComponentProperty.Uri] = {
+      value: {
+        stringValue: model.uri,
+      },
+    };
+  }
+  if (model.modelType) {
+    comp.properties![ModelRefComponentProperty.ModelType] = {
+      value: {
+        stringValue: model.modelType,
+      },
+    };
+  }
```

Comment:
`uri` and `modelType` are required field so no need to check if they exist. just set their value at line 21